update.php

```diff
@@ -1349,3 +1349,20 @@ function update_1524(): int
 
 	return Update::SUCCESS;
 }
+
+function update_1525(): int
+{
+	if (!DBA::e('UPDATE `user` u
+    JOIN `profile` p
+    ON p.`uid` = u.`uid`
+    SET u.`username` = p.`name`')) {
```

Comment:
I don't think that this is needed. For the owner view for example we are using the name from the self contact. And that is filled by the username. So I think that the username has the higher "priority" and shouldn't be overwritten by name in the profile.